ci-operator/step-registry/optional-operators/cvp-common/scorecard/optional-operators-cvp-common-scorecard-commands.sh

```diff
@@ -5,6 +5,43 @@ set -o errexit
 set -o pipefail
 set -x
 
+retry() {
+        local retries_max=3
+        local attempt_num=1
+        until "$@"
```

Comment:
Could change the condition here to `[[ ! $(jq . "${ARTIFACT_DIR}"/scorecard-output-basic.json) ]]` and skip checking it lower in the script? This would also mean that the loop would end if the operator failed the tests legitimately.